underpants.js

```diff
@@ -265,6 +458,31 @@ var _ = {};
 *   _.some([1,2,3], function(e){return e % 2 === 0}) -> true
 */
 
+// accepts a collection and a func
+// call function for every elem of collection with following parameters:
+//      if array, current elem, its index, collection
+//      if object, current val, its key, collection
+// if all elements are false, return false
+// if any are true, return true
+// if no function, return false if all falsy, otherwise true
+
+_.some = (col, func) => {
+    let hasTrue = false;
+    if (_.typeOf(func) !== "function") {
+        _.each(col, function (elem, index, collection) {
+            if (elem) {
+                hasTrue = true;
+            }
+        })
+    } else {
+        _.each(col, function (elem, index, collection) {
+            if (func(elem, index, collection)) {
+                hasTrue = true;
+            }
+        });
+    }
+    return (hasTrue) ? true : false;
```

Comment:
```suggestion
    return hasTrue;
```

This ternary statement is the equivalent of this code:

```js
if (XYZ === true) {
  return true; 
} else {
  return false;
}
```

If you ever find yourself in this pattern, you can simply `return XYZ`